lib/metanorma/standoc/requirement.rb

```diff
@@ -0,0 +1,65 @@
+module Metanorma
+  module Standoc
+    class Requirement
+      def satisfied
+        raise NotImplementedError("abstract method")
+      end
+
+      def cmd
+        raise NotImplementedError("abstract method")
+      end
+    end
+
+    class LatexmlRequirement < Requirement
+      def initialize
+        @recommended_version = '0.8.4'
+        @minimal_version = '0.8.0'
+        version_output, = Open3.capture2e("latexml --VERSION")
+        @actual_version = version_output&.match(%r{\d+(.\d+)*})
+      end
+
+      def satisfied
+        version = @actual_version
+
+        sanity_check(version)
+
+        if Gem::Version.new(version) < Gem::Version.new(@recommended_version)
+          version = "unknown" if version.to_s.empty?
+          header_msg = "WARNING latexmlmath version #{version} below 0.8.4!"
+          suggestion = if Gem.win_platform?
+                         "cmd encoding is set to UTF-8 with `chcp 65001`"
+                       else
+                         "terminal encoding is set to UTF-8 with `export LANG=en_US.UTF-8`"
+                       end
+
+          warn "#{header_msg} Please sure that #{suggestion} command"
+        end
+      end
+
+      def cmd
+        version = @actual_version
+
+        sanity_check(version)
+
+        if Gem::Version.new(version) < Gem::Version.new(@recommended_version)
```

Comment:
You mean `<=` for comparing versions, right?

Reply:
nope, because we use `--inputencoding=UTF-8` only for `0.8.4`

Reply:
Right, got confused 👍